descriptor/qsarmolecular/src/main/java/org/openscience/cdk/geometry/surface/NumericalSurface.java

```diff
@@ -189,6 +190,29 @@ private void init() {
         }
         return (ret);
     }
+    
+    public ArrayList<Point_Type> getAllPointswithAtomType() {
```

Comment:
1. Can you please add a JUnit test case for this new methods too?
2. Please provide JavaDoc to describe the functionality.
3. I think the return type can be just `List<Point_Type>`